core/src/test/java/org/wso2/carbon/testgrid/core/ScenarioExecutorTest.java

```diff
@@ -89,7 +93,7 @@ public void runScenarioTest() throws ScenarioExecutorException {
 
             whenNew(TestReportEngineImpl.class).withNoArguments().thenReturn(testReportEngine);
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error(e);
```

Comment:
Since an exception depicts a test failure in this test case, we should not catch this exception

Reply:
Will be fixed with #54